doc/user-manual/language/copatterns.lagda.rst

```diff
@@ -119,7 +122,33 @@ The resulting behaviour is the same in both cases:
     test₂ : backward-2 enum-Nat 5 ≡ 3
     test₂ = refl
 
+Copatterns with coinduction
+----------------------------------
+
```

Comment:
The option `--guardedness` which is necessary to process this example, should be mentioned in the text here.